synfig-studio/src/gui/duckmatic.cpp

```diff
@@ -462,7 +513,9 @@ Duckmatic::start_duck_drag(const synfig::Vector& offset)
 		duck_dragger_->begin_duck_drag(this,offset);
 
 	//drag_offset_=offset;
-	drag_offset_=find_duck(offset)->get_trans_point();
+	//in the case of using the select tool there are no ducks visible
+	if (find_duck(offset))
```

Comment:
It's better to store result in a variable so that the function is not called twice.